mobile/src/main/AndroidManifest.xml

```diff
@@ -24,6 +24,7 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
WRITE_EXTERNAL_STORAGE automatically grants READ permissions. You can remove this.

Reply:
For some reason I thought I read this had changed. I looked it up again, and apparently it didn't.